saleor/product/models.py

```diff
@@ -390,6 +388,7 @@ def __str__(self):
 
     @property
     def quantity_available(self):
+        self.refresh_from_db()
```

Comment:
We should rather make sure that the code that uses this property has always the fresh data.